sensor/common/metrics/metrics.go

```diff
@@ -20,6 +20,20 @@ var (
 		Help:      "Number of panic calls within Sensor.",
 	}, []string{"FunctionName"})
 
+	detectorDedupeCacheHits = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.SensorSubsystem.String(),
+		Name:      "detector_dedupe_cache_hits",
```

Comment:
By the https://prometheus.io/docs/practices/naming/, this name should end with `_total`, but I see the rest of metrics in this file does not follow the convention...

Reply:
Feel free to resolve comment after reading.